.github/workflows/ccruntime_e2e_amd.yaml

```diff
@@ -0,0 +1,56 @@
+name: ccruntime e2e tests amd sev
+on:
+  workflow_call:
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    name: operator tests
+    strategy:
+      fail-fast: false
+      matrix:
+        runtimeclass: 
+          - "kata-qemu-sev"
+          - "kata-qemu-snp"
+        instance: 
+          - "coco-ci-amd-rome-001"
+          - "coco-ci-amd-milan-001"
+        exclude:
+          - runtimeclass: "kata-qemu-snp"
+            instance: "coco-ci-amd-rome-001"
+          - runtimeclass: "kata-qemu-sev"
+            instance: "coco-ci-amd-milan-001"
+
+    runs-on: [self-hosted]
+    steps:
+      - name: Take a pre-action for self-hosted runner
+        run: |
+          if [ -f ${HOME}/script/pre_action.sh ]; then
```

Comment:
The pre-action script is not mine and I believe it's Choi's per my understanding. It would be used by both ccruntime_e2e_amd.yaml as well as the ccruntime_e2e.yaml.